taskcluster/ci/build-bundle/kind.yml

```diff
@@ -27,7 +27,7 @@ job-defaults:
     aab-artifact-template:
         type: file
         name: public/target.aab
-        path: '/builds/worker/checkouts/src/app/build/outputs/bundle/{variant}/app.aab'
+        path: '/builds/worker/checkouts/src/app/build/outputs/bundle/{variant}/app-{variant}.aab'
```

Comment:
CC @JohanLorenzo Is that all that is needed here?

Reply:
I'm sorry, I'm not sure to understand why this path is changed. Does this PR also change the APKs location? If so, you also need to update https://github.com/mozilla-mobile/reference-browser/blob/176202b13cb2006ca2ddb6ce668807479d1f8b4f/taskcluster/ci/build/kind.yml#L34

Reply:
Nope, the PR doesn't change the APKs location. The new AGP changes only the bundle name to app-{variant}.aab.